templates/_nav/_secondary.twig

```diff
@@ -55,12 +55,14 @@
                 {# Add the available extensions that have added a menu-item. #}
                 {% set extend_menu = app['menu.admin'].get('extend') %}
                 {% for extension in extend_menu.children %}
-                    {{ self.linkExt(
-                        extension.icon|default('fa:briefcase'), 
-                        extension.label, 
-                        extension.uri, 
-                        (extension.uri == app.request.pathInfo)
-                    )}}
+                    {% if extension.permission is null or isallowed(extension.permission) == true or extension.permission == 'everyone' %}
```

Comment:
Hmm, the permission should never be null because of https://github.com/bolt/bolt/blob/77bef69482bd6eac29d0029e2c6086390e0fcbac/src/Menu/MenuEntry.php#L175 and if the permission is `everyone` everyone is allowed, so I think this can be shortened to `{% if isallowed(extension.permission) %}`